billing/billing.proto

```diff
@@ -119,8 +126,7 @@ message Transaction {
      Type(SSD/HDD/NVMe/etc)
       }
   */
-  map<string, google.protobuf.Value> meta = 9;
-  Currency currency = 10;
```

Comment:
Why should this be removed?

Reply:
164f47db0a315468a1e37cc58bef678ff0b27148 is literally proto from currency pr

Reply:
yeah, but we had the currencies merged, that's why suspicious, wy the field is removed.
so should it or should it not be removed?

Reply:
I renamed some stuff in https://github.com/slntopp/nocloud/pull/482
So https://github.com/slntopp/nocloud-proto/commit/164f47db0a315468a1e37cc58bef678ff0b27148 should be included